src/pages/admin/settings/api-keys.vue

```diff
@@ -80,6 +83,26 @@ meta:
             required
           />
         </div>
+        <AppSelect
+          v-model="newApiKeyData.expiresInDays"
+          :label="t('apiKey.data.expires')"
+          :items="[
+            { id: 0, label: t('adminSettings.apikey.expires.never') },
+            ...[30, 60, 90].map((days) => ({
+              id: days,
+              label: t('adminSettings.apikey.expires.days', { n: n(days) }),
+            })),
+          ]"
```

Comment:
Maybe like this instead?

```ts
const generateExpiresInOptions = () => {
  const neverExpiresOption = { id: 0, label: t('adminSettings.apikey.expires.never') };
  const daysOptions = [30, 60, 90].map(days => ({
    id: days,
    label: t('adminSettings.apikey.expires.days', { n: n(days) }),
  }));

  return [neverExpiresOption, ...daysOptions];
}

const expiresInOptions = computed(generateExpiresInOptions);
```


```html
  <AppSelect
    v-model="newApiKeyData.expiresInDays"
    :label="t('apiKey.data.expires')"
    :items="expiresInOptions"
    :info-message="t('adminSettings.apikey.expiresHelp')"
    required
    class="mb-4"
  />
  ```

Reply:
I knew you'd call me up on this :smile:... I've been coding on my own for too long! I'll fix it